src/components/WorldLayers/UploadFile.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+
+import config from "../../config/config";
+import { connect } from "react-redux";
+import { IState } from "../../store";
+import { IWorld } from "../../interfaces/IWorld";
+import { IWorldLayer } from '../../interfaces/IWorldLayer';
+import { ITBAction } from '../../consts/action-types';
+import { WorldsActions } from '../../actions/world.actions';
+import { LayerService } from '../../services/LayerService';
+
+/* Prime React components */
+import 'primereact/resources/primereact.min.css';
+import 'primereact/resources/themes/omega/theme.css';
+import 'primeicons/primeicons.css';
+import 'font-awesome/css/font-awesome.css';
+import { FileUpload } from 'primereact/components/fileupload/FileUpload';
+
+export interface IPropsUploadFiles {
+    worldName: string,
+    world: IWorld,
+    updateWorld: (worlds: IWorld) => ITBAction,
+    getAllLayersData: () => void
+}
+
+class UploadFile extends React.Component {
+    props: IPropsUploadFiles;
+    url: string = `${config.baseUrlApi}/upload/${this.props.worldName}`;
+
+    componentDidMount() {
+        this.url = `${config.baseUrlApi}/upload/${this.props.worldName}`;
+        console.warn("upload file url: " + this.url);
```

Comment:
remove the warning

Reply:
not yet. this component is not complete yet.